network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/blog_index_callout_box.html

```diff
@@ -10,7 +10,7 @@
         {% endif %}
       </div>
     {% endif %}
-    <h2 class="tw-text-[40px] tw-h2-heading tw-font-normal tw-mb-2">{{ value.title }}</h2>
+    <h2 class="tw-text-4xl tw-h2-heading tw-font-normal tw-mb-2">{{ value.title }}</h2>
```

Comment:
Updating from fixed height at all screen sizes to a new size thats responsive, though about 6px smaller then original, per request of Nancy